.flake8

```diff
@@ -27,4 +27,4 @@ select = C,D,E,F,W,B,B950
 # No longer complies with PEP8 See example, here:
 # https://github.com/PyCQA/pycodestyle/issues/373
 # and here: https://github.com/psf/black/issues/315
-ignore = E203,E501,W503
+ignore = E203,E501,W503,E712
```

Comment:
Dev is going to want a see a comment explaining why this is necessary.

Reply:
Added a comment in this commit: [6dfad1b](https://github.com/cisagov/pe-reports/pull/282/commits/6dfad1b08195be4a60b71de96b227e0488931edc)